.github/workflows/release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+      - beta
+      - '+([0-9])?(.{+([0-9]),x}).x'
```

Comment:
I'm assuming this will allow us to create branches that correspond to older versions that we might want to patch for example. will we have to manually specify which version we publish to from these branches?

Reply:
@intcreator no that will be automatically determined from the branch name, `semantic-release` really is that awesome ;)